### DISCUSSION TITLE
My account got hacked and I got banned from the Discord server

### DISCUSSION THREAD

**Discussion by tatratram** [fairy-stockfish/Fairy-Stockfish#837] [Q&A] [answered]
Dear mr. Fichter,
in May of this year, my e-mail got hacked which resulted in losing control over my Discord account. The account started spamming some scam link and I got banned from your Discord server. I've since managed to regain control over my accounts and I'm interested in joining the server again.
Kind regards,
Tatratram

**Comment by ianfab** [comment, marked as answer]
Ok, I revoked the ban. I recommend to make use of 2FA to try to prevent this from re-occuring.
